tests/baselines/reference/user/bignumber.js.log

```diff
@@ -0,0 +1,11 @@
+Exit Code: 1
+Standard output:
+node_modules/bignumber.js/bignumber.d.ts(1613,18): error TS2370: A rest parameter must be of an array type.
```

Comment:
update on bignumber.js' side. It looks like they are just changing the d.ts to what they imagine Typescript syntax would be.

Reply:
I'm going to check out their history before merging.